deployment/README.md

```diff
@@ -1,379 +1,466 @@
+
+1. Finish Kubernetes cluster configuration and deployment
+    - Configure the remaining resources with Terraform, for example: 
+         ```bash
+         cd $GIT_ROOT/deployment/terraform/kubernetes/
+         terraform init && terraform plan
+         terraform apply
+         ```
+    - Set your `kubectl` credentials, for example:
+         ```bash
+         gcloud container clusters get-credentials "$PREFIX-$ENV-gke-cluster" \
+           --region=$LOCATION --project="$PREFIX-$ENV-apps"
+         ```
+    - Apply the pod security policies, for example:
+         ```bash
+         kubectl apply \
+           -f $GIT_ROOT/deployment/kubernetes/pod_security_policy.yaml \
+           -f $GIT_ROOT/deployment/kubernetes/pod_security_policy-istio.yaml
+         ```
+    - Apply all deployments, for example:
+         ```bash
+         kubectl apply \
+           -f $GIT_ROOT/study-datastore/tf-deployment.yaml \
+           -f $GIT_ROOT/response-datastore/tf-deployment.yaml \
+           -f $GIT_ROOT/participant-datastore/consent-mgmt-module/tf-deployment.yaml \
+           -f $GIT_ROOT/participant-datastore/enroll-mgmt-module/tf-deployment.yaml \
+           -f $GIT_ROOT/participant-datastore/user-mgmt-module/tf-deployment.yaml \
+           -f $GIT_ROOT/study-builder/tf-deployment.yaml \
+           -f $GIT_ROOT/auth-server/tf-deployment.yaml \
+           -f $GIT_ROOT/participant-manager-datastore/tf-deployment.yaml \
+           -f $GIT_ROOT/hydra/tf-deployment.yaml \
+           -f $GIT_ROOT/participant-manager/tf-deployment.yaml
+         ```
+    - Apply all services, for example:
+         ```bash
+         kubectl apply \
+           -f $GIT_ROOT/study-datastore/tf-service.yaml \
+           -f $GIT_ROOT/response-datastore/tf-service.yaml \
+           -f $GIT_ROOT/participant-datastore/consent-mgmt-module/tf-service.yaml \
+           -f $GIT_ROOT/participant-datastore/enroll-mgmt-module/tf-service.yaml \
+           -f $GIT_ROOT/participant-datastore/user-mgmt-module/tf-service.yaml \
+           -f $GIT_ROOT/study-builder/tf-service.yaml \
+           -f $GIT_ROOT/auth-server/tf-service.yaml \
+           -f $GIT_ROOT/participant-manager-datastore/tf-service.yaml \
+           -f $GIT_ROOT/hydra/tf-service.yaml \
+           -f $GIT_ROOT/participant-manager/tf-service.yaml
+         ```
+    - Apply the certificate and the ingress, for example:
+         ```bash
+         kubectl apply \
+           -f $GIT_ROOT/deployment/kubernetes/cert.yaml \
+           -f $GIT_ROOT/deployment/kubernetes/ingress.yaml
+         ```
+    - Update firewalls:
+        - Run `kubectl describe ingress $PREFIX-$ENV`
+        - Look at the suggested commands under "Events", in the form of "Firewall
+        change required by network admin"
+        - Run each of the suggested commands
+1. Check the [Kubernetes dashboard](https://console.cloud.google.com/kubernetes/workload) in your `{PREFIX}-{ENV}-apps` project to view the status of your deployment
```

Comment:
it would be a good idea to ask users to wait until all apps are healthy, since the rest of the steps require healthy K8s pods and application to successfully run.

Reply:
adding